v0/destinations/clickup/util.js

```diff
@@ -268,18 +223,17 @@ const retrieveCustomFields = async (listId, apiToken) => {
   );
 
   if (processedCustomFieldsResponse.status !== 200) {
-    throw new ApiError(
+    throw new NetworkError(
       `Failed to fetch available custom fields due to "${JSON.stringify(
         processedCustomFieldsResponse.response
       )}"`,
       processedCustomFieldsResponse.status,
       {
-        scope: TRANSFORMER_METRIC.MEASUREMENT_TYPE.API.SCOPE,
-        meta: getDynamicMeta(processedCustomFieldsResponse.status)
+        [tags.TAG_NAMES.ERROR_TYPE]: getDynamicMeta(
```

Comment:
getDynamicMeta -> is this name still relevant ?

Reply:
No. Fixed it.